chrysler_pacifica_ehybrid_s_2019/drivers.launch

```diff
@@ -45,39 +45,17 @@ If not using simulated drivers they are activated if the respective mock argumen
 
 
   <!-- Specific Drivers -->
-  <arg name="drivers" default="ssc_interface_wrapper lightbar_driver avt_vimba_camera"
+  <arg name="drivers" default="ssc_interface_wrapper lightbar_driver"
```

Comment:
Lightbar driver can be removed from here since it is included as a ros2 node in drivers.launch.py

Reply:
Also, need to double check but seems like mock_can, mock_radar, mock_camera and mock_roadway_sensor arguments don't have corresponding services in this config and can be removed

Reply:
Lightbar driver and the unused mock drivers have been removed